src/main/java/com/nike/cerberus/operation/core/CreateBaseOperation.java

```diff
@@ -57,127 +50,37 @@
+        final Map<String, String> parameters = cloudFormationObjectMapper.convertValue(baseParameters, typeReference);
 
-        final String stackId = cloudFormationService.createStack(cloudFormationService.getEnvStackName(uniqueStackName),
+        cloudFormationService.createStack(StackName.BASE.getFullName(environmentName),
                 parameters, ConfigConstants.BASE_STACK_TEMPLATE_PATH, true);
-
-        final StackStatus endStatus =
-                cloudFormationService.waitForStatus(stackId,
-                        Sets.newHashSet(StackStatus.CREATE_COMPLETE, StackStatus.ROLLBACK_COMPLETE));
-
-        if (StackStatus.CREATE_COMPLETE == endStatus) {
-            final Map<String, String> stackOutputs = cloudFormationService.getStackOutputs(stackId);
-            final BaseOutputs outputParameters =
-                    cloudformationObjectMapper.convertValue(stackOutputs, BaseOutputs.class);
-
-            logger.info("Stack creation complete, uploading data to the configuration bucket.");
-
-            // ORDER IS IMPORTANT!
-
-            // 1. First update the environment metadata with the new config bucket name.
-            environmentMetadata.setBucketName(outputParameters.getConfigBucketName());
-
-            // 2. Initialize the environment config.
-            configStore.initEnvironmentData();
-
-            // 3. Write the first configuration values to the config bucket.
-            configStore.storeAzs(azByIdentifier.get(1), azByIdentifier.get(2), azByIdentifier.get(3));
-            configStore.storeStackId(StackName.BASE, stackId);
-            configStore.storeConfigKeyId(outputParameters.getConfigFileKeyId());
-
-            // 4. Initialize the secrets config.
-            configStore.initSecretsData();
-
-            // 5. Write the first secret value to the config bucket once the config key id has been saved.
-            configStore.storeCmsDatabasePassword(dbMasterPassword);
-
-            logger.info("Uploading complete.");
-        } else {
-            final String errorMessage = String.format("Unexpected end status: %s", endStatus.name());
-            logger.error(errorMessage);
-
-            throw new UnexpectedCloudFormationStatusException(errorMessage);
-        }
     }
 
     @Override
     public boolean isRunnable(final CreateBaseCommand command) {
-        if (StringUtils.isNotBlank(environmentMetadata.getBucketName())) {
-            try {
-                final String stackId = configStore.getStackId(StackName.BASE);
-
-                if (cloudFormationService.isStackPresent(stackId)) {
-                    logger.warn("Operation has already run for the specified environment.");
-                    return false;
-                }
-            } catch (IllegalStateException ise) { //NOPMD
-                // Don't care, fall through.
-            }
-        }
-
         return true;
```

Comment:
This seems wrong to me, you should be checking if the stack already exists because if it does 

        cloudFormationService.createStack(StackName.BASE.getFullName(environmentName),
                parameters, ConfigConstants.BASE_STACK_TEMPLATE_PATH, true);

will fail.

Reply:
- [ ] Implement isRunnable